### PR TITLE
Create footer

### LINKED ISSUE #17: Footer

### PR DESCRIPTION
**Pull request by hapo-ducdv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -266,6 +266,59 @@ <h5 class="card-title">Java Tutorial</h5>
...
+                    <div class="">facebook.com/tuyen.dung.haposoft</div>
+                </div>
+
+                <a data-toggle="collapse" href="#collapseCall" role="button" aria-expanded="false" aria-controls="collapseCall">
+                    <i class="fas fa-phone-alt footer-contact-icon"></i>
+                </a>
+                <div class="collapse" id="collapseCall">
+                    <div class="">+84-85-645-9898</div>
+                </div>
+
```
Comment (hapo-huypv) [inline index.html]: thừa enterlne

File: index.html [hunk 2]
```diff
@@ -266,6 +266,59 @@ <h5 class="card-title">Java Tutorial</h5>
...
+                </nav>
+            </div>
+            <div class="col-md-4 col-5 d-flex align-items-center justify-content-center footer-contact">
+                <a data-toggle="collapse" href="#collapseFacebook" role="button" aria-expanded="false" aria-controls="collapseFacebook">
+                    <i class="fab fa-facebook-f footer-contact-icon"></i>
+                </a>
+                <div class="collapse" id="collapseFacebook">
+                    <div class="">facebook.com/tuyen.dung.haposoft</div>
+                </div>
+
```
Comment (hapo-huypv) [inline index.html]: thừa enterline

**Review by hapo-huypv — COMMENTED** [review]
lgtm

**Review by hapo-nghialuu — APPROVED** [review]
LGTM

**Pull request merged**